tests/utils_tests/test_datastructures.py

```diff
@@ -168,7 +168,7 @@ def test_create_with_invalid_key(self):
             CaseInsensitiveMapping([(1, '2')])
 
     def test_list(self):
-        self.assertEqual(sorted(list(self.dict1)), sorted(['Accept', 'content-type']))
+        self.assertEqual(sorted(self.dict1), sorted(['Accept', 'content-type']))
```

Comment:
Since dicts are ordered in Python 3.6+, the `list()` could be kept and the `sorted()`s removed, I think.